### PR TITLE
PSR-1402 Removal of back button on specified pages once clicking 'vie…

### PR DESCRIPTION
**Pull request by Arei1997**
…w previous version of XXX(open in new tab)'

### CODE REVIEW

File: app/controllers/nonsipp/otherassetsdisposal/ReportedOtherAssetsDisposalListController.scala [hunk 1]
```diff
@@ -184,15 +192,32 @@ class ReportedOtherAssetsDisposalListController @Inject()(
...
+    srn: Srn,
+    page: Int,
+    year: String,
+    current: Int,
+    previous: Int
+  ): Action[AnyContent] = identifyAndRequireData(srn, ViewOnlyMode, year, (current - 1).max(0), (previous - 1).max(0)) {
+    implicit request =>
+      val showBackLink = false
+      val viewOnlyViewModel = ViewOnlyViewModel(
+        viewOnlyUpdated = if (ViewOnlyMode == ViewOnlyMode && request.previousUserAnswers.nonEmpty) {
```
Comment (ccatosdevelopment) [inline app/controllers/nonsipp/otherassetsdisposal/ReportedOtherAssetsDisposalListController.scala:205]: I think there's a typo here as `ViewOnlyMode == ViewOnlyMode` will always evaluate as true, did you mean `mode == ViewOnlyMode`?

File: app/controllers/nonsipp/memberdetails/SchemeMembersListController.scala [hunk 2]
```diff
@@ -128,20 +131,16 @@ class SchemeMembersListController @Inject()(
                       previousUserAnswers,
                       pages.nonsipp.memberdetails.Paths.personalDetails
                     ) == Updated
-                    logger.info(s"""[ViewOnlyMode] Status for member details list is ${if (updated) "updated"
```
Comment (ccatosdevelopment) [inline app/controllers/nonsipp/memberdetails/SchemeMembersListController.scala:131]: It looks like the logging has been removed from this controller, but left in for the other controllers, may I ask why it's been removed here?

File: app/controllers/nonsipp/otherassetsdisposal/ReportedOtherAssetsDisposalListController.scala (hunk 1 above)
Comment (Arei1997) [reply app/controllers/nonsipp/otherassetsdisposal/ReportedOtherAssetsDisposalListController.scala:205]: Thats correct, It should be Mode==viewOnlyMode
That should help the test case !

File: app/controllers/nonsipp/memberdetails/SchemeMembersListController.scala (hunk 2 above)
Comment (Arei1997) [reply app/controllers/nonsipp/memberdetails/SchemeMembersListController.scala:131]: Yes, I added those in to debug- I forgot to remove the others

**Review by ccatosdevelopment — APPROVED** [review]

**Pull request merged**